src/rust/cryptography-openssl/src/poly1305.rs

```diff
@@ -0,0 +1,48 @@
+// This file is dual licensed under the terms of the Apache License, Version
+// 2.0, and the BSD License. See the LICENSE file in the root of this repository
+// for complete details.
+
+#[cfg(any(CRYPTOGRAPHY_IS_BORINGSSL, CRYPTOGRAPHY_IS_LIBRESSL))]
```

Comment:
Rather than putting this `cfg` on everything, does it make more sense to just `cfg` out the `pub mod` in `lib.rs`?

Reply:
Makes sense, fixed!